spec/system/page_title_spec.rb

```diff
@@ -1,106 +1,117 @@
 require 'rails_helper'
```

Comment:
I don't really the get the purpose of refactoring this spec (how is it related to govspeak prompts?) I prefer the other style, but also don't care enough to not approve....

Reply:
I needed to touch this because we have added a new page at the start and I thought that this would help readability. I also edited the format of the `doc` output. BTW I did spot we don't title the results page.